### DISCUSSION TITLE
query indexes

### DISCUSSION THREAD

**Discussion by ceejayoz** [spatie/laravel-notification-log#7] [Ideas]
In a large app, `latestFor()` queries have the potential to do a lot of table scans.

Looking at the current implementation, `$table->index(['notifiable_type', 'notifiable_id'])` and `$table->index(['created_at'])` indexes look like they should be worthwhile as a default.

Would a PR be accepted to add these? Or is Spatie policy to leave indexing up to the user?

**Comment by freekmurze** [comment]
I would accept a PR that adds these indexes by default.
